ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java

```diff
@@ -114,4 +124,9 @@ public void clickOnKebabMenuAction(String action) {
         this.getRootElement().$(Element.KEBAB_DROPDOWN_MENU).shouldBe(visible).$$(By.tagName("a"))
                 .shouldHave(CollectionCondition.sizeGreaterThanOrEqual(3)).findBy(exactText(action)).shouldBe(visible).click();
     }
+
+    public void selectTab(String tabName) {
```

Comment:
The same could be achieved by `clickLink()` method?

https://github.com/syndesisio/syndesis-qe/blob/d35dc097cb635c680f6f07a4af6445c0ec75ad26/ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java#L164-L167